ruby-test-mode.el

```diff
@@ -69,49 +70,92 @@ Test Driven Development in Ruby."
   "The keymap used in `ruby-test-mode' buffers.")
 
 (defcustom ruby-test-file-name-extensions
-  '("builder" "erb" "haml" "rb" "rjs")
+  '("builder" "erb" "haml" "rb" "rjs" "rake" "slim")
```

Comment:
@zw963 Is it common to have tests for those extensions? If I remember there used to be tests for erb, haml and builder files, but I'm not sure. If that's still the case I'm fine with this.